### DISCUSSION TITLE
Problem with setting up bed / negative coordinates

### DISCUSSION THREAD

**Discussion by wdonders** [supermerill/SuperSlicer#3114] [Q&A]
For years I've been using Simplify3D and recently starting to look into Superslicer.
I've got one major no-go at this point: I can't get my bed setup properly!

My bed is 283 x 221 mm 
With the bottom left of the bed as 0,0, the far left bottom the print head can go is -62, -27,50 
(this negative area houses f.e. the nozzle wiper)

I did not change my Marlin settings, cause it works flawlessly with Simplify3D for years (see [marlin.jpg](https://ibb.co/S7f2Q05) )
It looks like I can setup the bed okay in SS, see [superslicer.jpg](https://ibb.co/twG8mjw)
However, a print placed in the center in SS, does not print in the center, see [octoprint.jpg](https://ibb.co/TMGXgsy)
The settings in Octoprint for my bed, see [octoprint2.jpg](https://ibb.co/LRdXTsW)

Who can help me in the right direction? For now, this a no-go for using Superslicer for me :-(

**Comment by mathiasludvigsson** [comment]
Did you try to set the origin to 0, 0 instead? In the picture from your octoprint the object is placed 62, 27.5mm off center.

**Reply by wdonders** [reply]
Jeezz, feel a little dumb now.. That did fix it!
So this setting has nothing to do with coordinates outside the normal bed coordinates.
Thanks!
